packages/gasket-log/src/index.d.ts

```diff
@@ -0,0 +1,104 @@
+/**
+ * Log instance that tracks fluentd connections and sets up a winston logger.
+ *
+ * @class Log
+ * @public
+ */
+export default class Log {
+    /**
+     * @constructor
+     * @param {object} [options] - Options
+     * @param {string} [options.level] - Default level to use
+     * @param {boolean} [options.silent] - Should logs be silenced
+     * @param {boolean} [options.local] - Is this for the local development
+     * @param {string} [options.prefix] - Message prefix to use
+     */
+    constructor(options?: {
+        level?: string;
+        silent?: boolean;
+        local?: boolean;
+        prefix?: string;
+    });
+    /** @private */
+    private options;
```

Comment:
You'd might as well leave out all the `private` members since in theory consumers don't need to know about them.